recipes/CommonVoice/ASR/CTC/extra_requirements.txt

```diff
@@ -1 +1 @@
-transformers==4.13
```

Comment:
move all extras to latest dependencies.

It's a testing convenience issue – otherwise each recipe will need their own env... these tests should be done at every release version - for the next major release, I'd like to either lift that freezing or freeze it to their latest version if possible. 

I get that just leaving it there is convenient for the end-user; it's not for maintenance through testing. On the other hand, rewriting the testing scripts for changing envs per recipe - why not (either) ...